ui/src/actions/__tests__/authors.test.js

```diff
@@ -96,6 +96,78 @@ describe('AUTHOR - async action creators', () => {
       done();
     });
 
+    it('and queries publications with proper size when user is cataloger', async done => {
+      mockHttp
+        .onGet('/literature?author=Harun&page=1&q=test&size=100')
+        .replyOnce(200, { foo: 'bar' });
+
+      const expectedActions = [
+        {
+          type: AUTHOR_PUBLICATIONS_REQUEST,
+          payload: { size: 100, page: 1, q: 'test', author: ['Harun'] },
+        },
+        { type: AUTHOR_PUBLICATIONS_SUCCESS, payload: { foo: 'bar' } },
+      ];
+
+      const store = getStoreWithState({
+        authors: fromJS({
+          data: {
+            metadata: {
+              facet_author_name: 'Harun',
+            },
+          },
+          publications: {
+            query: { author: ['Harun'] },
+          },
+        }),
+        user: fromJS({
+          loggedIn: true,
+          data: {
+            roles: ['cataloger'],
+          },
+        }),
+      });
+      await store.dispatch(fetchAuthorPublications({ page: 1, q: 'test' }));
+      expect(store.getActions()).toEqual(expectedActions);
+      done();
+    });
+
+    it('and queries publications with proper size when user is not cataloger', async done => {
```

Comment:
this isn't needed.

Reply:
We don't want to be sure that normal users get 10 results and not 100?

Reply:
the existing test is already testing this, no?

Reply:
No, we're not. Every other test in authors.test.js is using `size=5` as specified in test payload. None of them are using default `10`

Reply:
default doesn't matter, it's not concern of the the action here, but the reducer. So as long as we're testing the size is not overwritten with `100` for non catalogers, that's OK.

Reply:
:+1: